lisp/ess-inf.el

```diff
@@ -125,6 +125,11 @@ If `ess-plain-first-buffername', then initial process is number-free."
                         (= n 1))) ; if not both first and plain-first add number
               (concat ":" (number-to-string n)))))
 
+(defvar-local inferior-ess--start-name nil
+  "Program name used to start the inferior process.")
```

Comment:
Shall we put this into one internal variable?

Reply:
Done.